dataModel/SsoTables.sql

```diff
@@ -10,43 +10,41 @@
 -- https://github.com/fluid-project/personal-data-server/blob/main/LICENSE
 
 -- SSO Provider
-CREATE TABLE "AppSsoProvider" (
-    "providerId" SERIAL NOT NULL PRIMARY KEY,
+CREATE TABLE "sso_provider" (
+    "provider_id" SERIAL NOT NULL PRIMARY KEY,
     "provider" varchar(30) NOT NULL,
     "name" varchar(40) NOT NULL,
     "client_id" varchar(191) NOT NULL,
-    "client_secret" varchar(191) NOT NULL
+    "client_secret" varchar(191) NOT NULL,
+    "created_timestamp" TIMESTAMPTZ NULL,
+    "last_updated_timestamp" TIMESTAMPTZ NULL
 );
 
--- User
-CREATE TABLE "User" (
-    "userId" VARCHAR(64) PRIMARY KEY NOT NULL,
-    name VARCHAR(64) NOT NULL,
-    username VARCHAR(64) NOT NULL,
-    derived_key VARCHAR(255) NOT NULL,
-    verification_code VARCHAR(255) NOT NULL,
-    salt VARCHAR(255) NOT NULL,
-    iterations INT NOT NULL,
-    email VARCHAR(32) NOT NULL,
-    roles VARCHAR(16)[] NOT NULL,
-    verified BOOLEAN NOT NULL DEFAULT false
+-- user
+CREATE TABLE "local_user" (
```

Comment:
Is this quite right? Didn't we think that "local_user" would actually be a properly local user, that is, it would hold the info for local authentication of the user, e.g. the salt, verification code, etc. 
The documentation still references this table as "user" and I think that is probably right.